recipes/f77_zmq/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "f77_zmq" %}
+{% set version = "4.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zeromq/f77_zmq/releases/download/v4.3.2_autotools/f77-zmq-4.3.2.tar.gz
+  sha256: e027bf1017ca6b47ce4a3481ae5b255db8ea86294bc32f1207bd90dbdece7671
+
+build:
+  skip: true  # [win]
+  number: 0
```

Comment:
This is a library that others link against, so it should have a `run_export` that uses the `pin_compatible( name|lower , max_pin=x)` syntax to pin the compatible versions.

Reply:
Please address this comment.

Reply:
Done